util/config/make_sys_basic_types.py

```diff
@@ -127,6 +127,15 @@ def get_sys_c_types(docs=False):
         elif 'FIND_INT_SIZES_START' in line:
             keep = True
 
+        # Ignore lines starting with # since they could be #line
+        # type directives.
+        elif line.startswith("#"):
```

Comment:
Could there be whitespace between the start of the line and the `#`? If so, you might change this to: `line.strip().startswith('#')`.